plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiMetadata.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Please update `TestHudiConnectorTest` for testing time travel with query. 

There is no need to add `TestHudiMetadata` in this PR. It's not so helpful.